package.json

```diff
@@ -35,7 +36,6 @@
     "build": "react-scripts build",
     "format": "prettier --write .",
     "generate-contract-types": "./scripts/generate-contract-types.sh",
-    "generate-proto-types": "./scripts/generate-proto-types.sh -ssh",
     "lint": "eslint --fix --ext .ts,.tsx .",
     "postinstall": "npm run generate-contract-types && npm run generate-proto-types",
```

Comment:
`&& npm run generate-proto-types` should be now removed

Reply:
Right! Done